modules/internal/ArrayViewSlice.chpl

```diff
@@ -177,9 +177,10 @@ module ArrayViewSlice {
 
     iter these(param tag: iterKind) ref
       where tag == iterKind.standalone && !localeModelHasSublocales &&
-           __primitive("method call resolves", privDom, "these", tag) {
-      const ref myarr = arr;
-      forall i in privDom do yield myarr.dsiAccess(i);
+      __primitive("method call resolves", privDom, "these", tag) {
+        //        writeln("IN slice standalone iterator");
```

Comment:
Looks like debugging code to be removed.

Reply:
Yeah, this is just a draft PR, waiting to determine whether we can proceed with it or not (where the performance mystery had been the blocking factor until learning tonight that comms didn't change in counts, just location).